exercises/minesweeper/description.md

```diff
@@ -1,25 +1,35 @@
-Add the numbers to a minesweeper board.
+Add the score totals to a completed Minesweeper board.
```

Comment:
I wasn't sure it was right to call them "score totals" since I haven't seen a case where the numbers on the board contribute to scoring; usually I saw score counted in time taken or number of steps taken.

Perhaps I could suggest "mine counts" if "numbers" is deemed unsuitable. I would consider "numbers" suitable.

Reply:
I'm going to split the difference and go with "mine counts" ... "numbers" is ambiguous.

Reply:
Fixed